feature-configs/cn-ran-overlays/README.md

```diff
@@ -0,0 +1,22 @@
+# cn-ran-overlays
+
+This folder contains an example configurations for 5G DU site
+(CU - work in progress)
+
+
+## Overview
+
+The [`all-in-one`](all-in-one) directory contains the Kustomize profile for deployment of DU integration features, namely:
+- SCTP MachineConfig patch
+- Performance addon operator and DU performance profile
+- PTP operator and slave profile
+- SR-IOV operator and associated profiles
+
+## Deployment
+
+1. Make sure your nodes are labeled as required and MCP is created. If your cluster is brand new, this can be done by cnf-features-deploy/hack/setup-test-cluster.sh
+2. Deploy using the usual project tools:
+
+  `FEATURES_ENVIRONMENT=cn-ran-overlays FEATURES=all-in-one make feature-deploy`
+
+3. Wait for the iterations to complete
```

Comment:
that's what make feature-wait is about
which
a. make it explicit
b. enable notification of failure

@fedepaol maybe it make sense to add a Makefile target that does feature-deploy + feature_wait (without setup and functests).

Reply:
Don't think so. feature-wait serves CI.
If a user wants to use the features, he just deploys and apply the configuration.

Reply:
ok
but my point here is also that 
we can't CI the thing without a working feature-wait

Reply:
Yep, but for CI you also need to do setup and functests  (see my comment below)